test/e2e/suite_test.go

```diff
@@ -226,13 +226,13 @@ func TestOperator(t *testing.T) {
 	}))
 	t.Run("CreateSingleMedusaJob", e2eTest(ctx, &e2eTestOpts{
```

Comment:
Probably unrelated, but why some of the medusa tests are called "Old" and others are called "Job"?

Reply:
In 1.1, we introduced a new set of CRDs named MedusaBackup, MedusaBackupJob and MedusaRestoreJob with the associated controllers. The old APIs are deprecated but still usable, hence the additional tests. They were renamed with the `Old` suffix because invoking the e2e tests with just  `TestOperator/CreateSingleMedusa` would also trigger `CreateSingleMedusaJob` (that's the way subtests work I guess, it just sees what you enter as a prefix).